lib/ReactViews/Map/Panels/HelpPanel/help-panel.scss

```diff
@@ -1,15 +1,15 @@
-@import "~terriajs-variables";
-@import "../../../../Sass/common/mixins";
+@use "../../../../Sass/common/_variables";
+@use "../../../../Sass/common/mixins";
```

Comment:
Some of these files import `mixins`, and others import `_mixins` (see `guidance-dot.scss` vs this file, both in subdirectories of `ReactViews`), is that intentional?